index.html

```diff
@@ -541,11 +541,14 @@ <h1>{{ chosen_domain == 'US' ? 'USA' :
   scale = scale || 'log10';
   if (chart) { chart.detach(); }
   if (stat == 'deltas') {
```

Comment:
maybe allow the stat option string to specify the window width.  E.g.,
`
if (stat.match(/^\d+day$/)) {
  const deltaInterval = parseInt(stat.match(/^(\d+)day$/)[0]);
`

Reply:
That's only worthwhile if we let the user change the size. And if we do, maybe it would feel cleaner to do it via a separate `chosen_delta_interval` variable to avoid having to parse out the delta option string?

I think I'll leave that out initially, and we can revisit. As long as the code is flexible on the interval size, we keep our options open.